resources/js/features/events/components/EventAwardTiers/AwardTierItem.tsx

```diff
@@ -0,0 +1,115 @@
+import type { FC } from 'react';
+import { useTranslation } from 'react-i18next';
+import { LuCheck } from 'react-icons/lu';
+
+import {
+  BaseTooltip,
+  BaseTooltipContent,
+  BaseTooltipTrigger,
+} from '@/common/components/+vendor/BaseTooltip';
+import { cn } from '@/common/utils/cn';
+import { formatDate } from '@/common/utils/l10n/formatDate';
+
+import { cleanEventAwardLabel } from '../../utils/cleanEventAwardLabel';
+
+interface AwardTierItemProps {
+  event: App.Platform.Data.Event;
+  eventAward: App.Platform.Data.EventAward;
+}
+
+export const AwardTierItem: FC<AwardTierItemProps> = ({ event, eventAward }) => {
+  const { t } = useTranslation();
+
+  const earnersMessage = useEarnersMessage(!!eventAward.earnedAt, eventAward.badgeCount!);
+
+  // Clean the award label by removing the event title prefix if present.
+  const cleanedAwardLabel = cleanEventAwardLabel(eventAward.label, event);
```

Comment:
> I think after this is merged it may be best to put a character limit on event award titles and append the event title where necessary, rather than having UI routines to remove it.

I never expected the event titles to appear in the awards names.